src/util/util.js

```diff
@@ -93,4 +283,172 @@ export function getEnvs() {
   };
 }
 
-export const queryPatientIdKey = 'launch_queryPatientId';
+export function scrollToAnchor(anchorElementId) {
+  const targetElement = document.querySelector(
+    `#${QUESTIONNAIRE_ANCHOR_ID_PREFIX}_${anchorElementId}`
+  );
+  if (!targetElement) return;
+  targetElement.scrollIntoView();
+}
+
+export function getElmDependencies() {
+  const elmJsonDependencyArray = [commonLibrary];
+  // Reformat ELM JSON value set references to match what is expected by the
+  // code service built into the cql execution engine
```

Comment:
What's the underlying issue here? Do our FHIR resources use absolute URLs instead of relative URLs?